scrapers/src/extractors/party.rs

```diff
@@ -0,0 +1,85 @@
+use std::sync::OnceLock;
+
+use regex::Regex;
+
+static MATCHERS: OnceLock<Vec<(Regex, &'static str)>> = OnceLock::new();
+
+pub fn extract_party_name(input: &str) -> Option<String> {
+    let matchers = MATCHERS.get_or_init(|| {
+        [
+            (
+                r"(?i:(?:Unaffiliated|No Party Affiliation))",
+                "Unaffiliated",
+            ),
+            (r"(?i:Democratic Party)", "Democratic Party"),
+            (r"(?i:Republican Party)", "Republican Party"),
+            (r"(?i:Libertarian Party)", "Libertarian Party"),
+            (
+                r"(?i:American Constitution Party)",
+                "American Constitution Party",
+            ),
+            (r"(?i:Center Party)", "Center Party"),
+            (r"(?i:Unity Party)", "Unity Party"),
+            (r"(?i:Forward Party)", "Forward Party"),
+            (r"(?i:Approval Voting Party)", "Approval Voting Party"),
+        ]
```

Comment:
as we add additional sources, we could expand these matchers here to fit new data right?

Reply:
Correct, this currently only encompasses parties I've seen in the CO data. This is obviously missing ones like Independents and the various Green parties.